pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -0,0 +1,184 @@
+To add custom catalog applications to a Project, a GitRepository pointing to the catalog Git repository **must be also created on each attached cluster in the Project**. Follow the steps on the [Create a Git Repository][project-custom-applications-git-repo] page, but apply the same commands on each attached cluster that is in the Project.
+
+```sh
+kubectl apply -f - <<EOF
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: example-repo
+  namespace: <project-namespace>
+spec:
+  interval: 1m0s
+  ref:
+    branch: <your-target-branch-name> # e.g., main
+  timeout: 20s
+  url: https://github.com/<example-org>/<example-repo>
+EOF
+```
+
+### Create cert-manager resources on clusters with cert-manager pre-installed prior to attaching them
+
+If you are attaching a cluster that already has `cert-manager` installed, you need to manually create some cert-manager resources prior to attaching your cluster.
+
+For example, [Konvoy-created clusters that are self-managed][konvoy-self-managed] have `cert-manager` already installed to the `cert-manager` namespace.
+
+Create the following yaml file:
+
+```yaml
+cat << EOF > cert_manager_root-ca.yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: kommander-bootstrap-ca-issuer
+  namespace: cert-manager
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: kommander-bootstrap-root-certificate
+  namespace: cert-manager
+spec:
+  commonName: ca.kommander-bootstrap
+  dnsNames:
+    - ca.kommander-bootstrap
+  duration: 8760h
+  isCA: true
+  issuerRef:
+    name: kommander-bootstrap-ca-issuer
+  secretName: kommander-bootstrap-root-ca
+  subject:
+    organizations:
+      - cert-manager
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: kommander-bootstrap-issuer
+  namespace: cert-manager
+spec:
+  ca:
+    secretName: kommander-bootstrap-root-ca
+EOF
+```
+
+Then, apply this file to your cluster:
+
+```sh
+kubectl apply -f cert_manager_root-ca.yaml
+```
+
+Once complete, continue to [attach your cluster][attach-cluster] to Kommander.
+
+You should expect to see that cert-manager will fail to deploy due to your existing `cert-manager` installation. This is expected and can be ignored.
+
+### New features and capabilities
+
+#### DKP Major Version Upgrade
+
+Konvoy and Kommander 2.1 represent a major version upgrade that moves forward DKP architecture to give you access to D2iQ's next generation centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.1 enhances your existing clusters to use a new architecture.
```

Comment:
Don't need this here bc you can't major version upgrade to 2.1.1. We WILL need a note to install fresh with 2.1.1.

Reply:
I'm a little confused (so what's new??? :-) )
This text (and review comment) is for this file:

pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

Which is just a copy of the 2.1 RN.

The 2.1.0 release notes were originally in pages/dkp/kommander/2.1/release-notes/index.md, now because we have a patch release, they need to move to pages/dkp/kommander/2.1/release-notes/2.1.0/index.md, and the prevew 'release-notes/index.md' becomes a 'placeholder page', right?

So, I think we shouldn't be removing anything here because this isn't 2.1.1 content, this is already released content, just moved to a new file, right?

Reply:
I think we decided you can major version upgrade to 2.1.1 from 1.8.4.  @kevinmiller-conley and co have it on their radar to validating that, I believe.  Was also confirmed by @samvantran here:  https://mesosphere.slack.com/archives/C01QCJ3V16X/p1638819274049500?thread_ts=1638816254.045900&cid=C01QCJ3V16X.  You just can't upgrade to 2.1.1 from 2.1.0 or 2.0.0 so if you are already on a 2.x series, it's a fresh install.  Ehhh so many pathways! ha

Reply:
I'm sorry I didn't get the message on that. We'll need to figure out this NEW docs requirement.